### ISSUE TITLE
Guild members not being downloaded when using GatewayIntents

### ISSUE THREAD

**Issue by NoiTheCat** [RogueException/Discord.Net#1646]
With `AlwaysDownloadUsers = true` and subscribing to `GatewayIntents.GuildMembers`, the bot appears to disregard the former and does not automatically download and cache all members when connecting to large guilds. The issue is solved by not using GatewayIntents at all within the client options.

**Comment by SubZero0** [comment]
I think I know why that's happening and #1647 will probably fix it, could you test it?
Otherwise I'll try to check it later (a few days, maybe 1-2 weeks), need to know if I'll need to change the type.

**Comment by NoiTheCat** [comment]
Sorry for the delay! I gave your change a try and the issue is still there, I'm afraid. The user cache for each guild remains mostly empty, even after a few minutes of letting the bot sit there (not sure if that matters, but I did try it).

**Comment by foxbot** [comment]
Downloading users with intents enabled are no longer a Discord supported use-case, and I would strongly encourage you to move your bot away from this requirement. If you feel that your bot requires downloading users, I would suggest directing your feedback to Discord's Bot API team. The underlying bug will be resolved in #1647 however.

**Comment by NoiTheCat** [comment]
> Downloading users with intents enabled are no longer a Discord supported use-case

I'm not sure what you mean by this. Discord still allows bots to download users, it has just changed from batch downloading to only allowing one server per request. The functionality is certainly still there as stated in their [docs](https://discord.com/developers/docs/topics/gateway#request-guild-members), and I've noticed at least one other library, discord.py, implement equivalent functionality with their `chunk_guilds_at_startup` option.

As I stated earlier though, the patch given above seemed to have no effect when I tested it for myself.
